.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI workflow
```

Comment:
Could you update the readme accordingly?

```
![CI workflow](https://github.com/fastify/fastify-env/workflows/CI%20workflow/badge.svg)
```

Reply:
Sure, updated